code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -361,65 +361,68 @@
 
 	return
 
+/mob/living/simple_animal/bot/medbot/proc/assess_beaker_injection(mob/living/carbon/C)
+	//If we have and are using a medicine beaker, return any reagent the patient is missing
+	if(use_beaker && reagent_glass && reagent_glass.reagents.total_volume)
```

Comment:
```suggestion
	if(use_beaker && reagent_glass?.reagents.total_volume)
```

Reply:
Done, as well as an exact same check a bit lower in the file